.github/workflows/ci.yml

```diff
@@ -40,6 +40,10 @@ jobs:
       - name: Lint Ruby files
         run: bundle exec rubocop
 
+      - name: Lint RBI shims
+        if: matrix.os != 'windows-latest'
```

Comment:
I'm unsure why this fails on Windows:
https://github.com/Shopify/ruby-lsp/actions/runs/6723189944/job/18272722122

Reply:
> We don't install Tapioca, Sorbet, Spoom, etc on Windows because Sorbet is no compatible. That's likely why.

But how would that result a `Layout/EndOfLine` failure?

> I think this should be a separate job from CI. There's no reason why we would check shims for every OS and every Ruby version.

Yeah, I think we can avoid of lot of those runs. I've updated `ci.yml` to split the job into two (it might be better to have this in a separate PR though?).

Reply:
Maybe we lint the shims and that fails because on Windows the end of line is expected to be `\r\n` rather then `\n`. Not sure.